WordPress/src/main/java/org/wordpress/android/ui/WPWebViewActivity.java

```diff
@@ -254,6 +254,10 @@ protected void configureWebView() {
             }
             webViewClient = new WPWebViewClient(site, mAccountStore.getAccessToken(), allowedURL);
         } else {
+            if (allowedURL == null || allowedURL.size() == 0) {
```

Comment:
The `allowedURL` is not the url user is trying to browse, it's just a list we keep if we wanted to disable most of the links in the webview, so the user doesn't do weird stuff from it and blame the app. If we don't want to disable the links `allowedURL` would be null and this would finish the activity.

I am not sure what the intentions behind this change were but I think there was a misunderstanding about how this works. As an example, if you try to view a blog post's preview from the post list, it'll just not work. Hope that makes sense!